docs/semgrep-code/remove-duplicates.md

```diff
@@ -5,26 +5,116 @@ title: Remove duplicate findings
 hide_title: true
 description: Learn how to remove duplicate findings and prevent them from displayed in Semgrep AppSec Platform.
 tags:
-  - Semgrep Code
-  - Semgrep AppSec Platform
+ - Semgrep Code
+ - Semgrep AppSec Platform
 ---
 
 # Remove duplicate findings
 
 Semgrep scans are performed on both mainline (trunk) and non-mainline branches. The scope of the scan can differ depending on if Semgrep is called on a mainline or non-mainline branch.
 
 <dl>
-    <dt>Full scan</dt>
-    <dd>Scans the repository in its entirety. It is recommended to perform full scans on mainline branches, such as <code>master</code> or <code>main</code>. This scan is performed on a scheduled basis.</dd>
-    <dt>Diff-aware scan</dt>
-    <dd>Diff-aware scans are performed on non-mainline branches, such as in pull requests and merge requests. Diff-aware scans traverse the repository's files based on the commit where the branch diverged from the mainline branch (or diverged from the last commit that was fully scanned)</dd>
+ <dt>Full scan</dt>
+ <dd>Scans the repository in its entirety. It is recommended to perform full scans on mainline branches, such as <code>master</code> or <code>main</code>. This scan is performed on a scheduled basis.</dd>
+ <dt>Diff-aware scan</dt>
+ <dd>Diff-aware scans are performed on non-mainline branches, such as in pull requests and merge requests. Diff-aware scans traverse the repository's files based on the commit where the branch diverged from the mainline branch (or diverged from the last commit that was fully scanned)</dd>
 </dl>
 
-## Remove duplicate findings using Semgrep AppSec Platform
+## How Semgrep distinguishes between new and duplicate findings
 
-Regardless of the scope of a scan, Semgrep correlates findings across branches based on their unique fingerprint, automatically deduplicating findings and making it simpler to triage.
+Semgrep generates a finding whenever it scans a project, and one of its rules matches a piece of code in the project. Since Semgrep scans a project multiple times, it needs a way to track the same finding in a file over time. Semgrep does this using two types of fingerprints: `syntactic_id` and `match_based_id`.
 
-If a finding is fixed in one branch (such as `main`) but open in another (such as `production`), and the code fixes are present in both branches, initiate a scan through your CI job or SCM tool on the branch(es) with open findings to have Semgrep mark the findings as fixed.
+
+:::info
+The calculations used to determine whether findings are new are subject to change at any time as Semgrep improves its deduplication logic.
+:::
+
+### `syntactic_id`
```

Comment:
I would +1 to Alexis's comment to talk about `match_based_id` first. While we do still expose `syntactic_id` as a field in the response of the public API endpoint to retrieve a list of issues, and this is still used on the engine to do some sort of deduplication there (this is totally separate from the platform-side deduplication just to be clear), I think for the user, this information might not be as important/relevant as `match_based_id`. 

Even for us internally, it's mostly used to indicate if a finding closer to an _exact_ match to another finding, but we still use `match_based_id` as the source of truth for platform side deduplication.